lizzy_client/lizzy.py

```diff
@@ -77,8 +77,18 @@ def get_stacks(self) -> list:
         request.raise_for_status()
         return request.json()
 
-    def new_stack(self, image_version: str, keep_stacks: int, new_traffic: int, senza_yaml_path: str,
-                  application_version: Optional[str], disable_rollback: bool, parameters: List[str]) -> str:
+    def new_stack(self,
+                  image_version: str,
+                  keep_stacks: int,
+                  new_traffic: int,
+                  senza_yaml_path: str,
+                  stack_version: Optional[str],
```

Comment:
Yes, it can be `None` if omitted on the command line, in which case it will not be added to the request data (see lines 107-109).